### PR TITLE
WIP: Github LFS Support

### PR DESCRIPTION
**Pull request by oogali**

### CODE REVIEW

File: src/ShareRoutes.test.jsx [hunk 1]
```diff
@@ -15,10 +15,10 @@ const tests = [
   {s: 'http://github.com/org/repo/blob/branch/a/b/c/file.ifc', out: pathBlobAbc},
   {s: 'https://www.github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
   {s: 'https://github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
-  {s: 'github.com/org/repo/blob/branch/file.ifc', out: pathBlob},
```
Comment (pablo-mayrgundter) [inline src/ShareRoutes.test.jsx:18]: The use case here is copying text from apps/sites that hide the protocol.  It's a nice-to-have

Comment (pablo-mayrgundter) [reply src/ShareRoutes.test.jsx:18]: I don't know.  I feel like server will upgrade to ssl if it wants and we don't have any private data per se, so http for me

File: src/utils/urls.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
+ * Gets the final URL needed to load an IFC model
```
Comment (pablo-mayrgundter) [inline src/utils/urls.js:2]: So am I understanding correctly that large files are hosted on a separate server, and this code detects the size difference and redirects the query to that server?

Comment (oogali) [reply src/utils/urls.js:2]: This code grabs the first 64 bytes of the file.

If that initial content contains a LFS signature, then it'll download from the Github LFS servers.

**Review by pablo-mayrgundter — APPROVED** [review]
Ok, LGTM!  Please extend the comment to describe the issue and mechanism a bit more in the util comment.

**Pull request closed without merging**